probe-rs/src/architecture/arm/ap/memory_ap/mock.rs

```diff
@@ -19,15 +19,21 @@ pub struct MockMemoryAp {
 impl MockMemoryAp {
     /// Creates a MockMemoryAp with the memory filled with a pattern where each byte is equal to its
     /// own address plus one (to avoid zeros). The pattern can be used as a canary pattern to ensure
-    /// writes do not clobber adjacent memory. The memory is also quite small so it can be feasibly
-    /// printed out for debugging.
+    /// writes do not clobber adjacent memory.
```

Comment:
I disagree with the old comment, 2x32K isn't smal enough to be printed.

Reply:
I guess this was bumped after the comment :D